ledger/src/snapshot_utils.rs

```diff
@@ -153,14 +247,22 @@ pub fn add_snapshot<P: AsRef<Path>>(snapshot_path: P, bank: &Bank) -> Result<()>
         snapshot_file_path,
     );
 
-    let snapshot_file = File::create(&snapshot_file_path)?;
-    // snapshot writer
-    let mut snapshot_stream = BufWriter::new(snapshot_file);
     // Create the snapshot
-    serialize_into(&mut snapshot_stream, &*bank)?;
     let mut bank_rc_serialize = Measure::start("create snapshot");
```

Comment:
This `Measure` will start to include the processing time of [serializing a Bank](https://github.com/solana-labs/solana/pull/7182/files#diff-0f0af8fcfa1444e1b07d671c5bfa4a69R254), in addition to original `BankRc`. I think including it isn't a problem because those serializations (`Bank` and `BankRc`) are tightly coupled.

@carllin  Is there any concern for this measurement definition change? (I'm asking based on [the original author of this `Measure`](https://github.com/solana-labs/solana/pull/6047/files#diff-ff88facee39736079f15b8937943a82eR144))